name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/SecurityListView.java

```diff
@@ -113,13 +113,19 @@ public void menuAboutToShow(IMenuManager manager)
                         @Override
                         public String getText(Object element)
                         {
+                            if (element == null)
+                                return null;
```

Comment:
How can this element be null?!?

Reply:
äh... i think i have an other problem...
https://bugs.eclipse.org/bugs/show_bug.cgi?id=570553

Please remove this lines

Reply:
The variable `query` in the CoinGecko search provider is null and thus an error is generated, causing "`item.getSymbol()`" to generate an error.

Reply:
I can't reproduce the error again at this time, but it was written to the error log on my end.
Currently, however, this is flooded by an error after update of Eclipse. I would have made a screenshot.
https://bugs.eclipse.org/bugs/show_bug.cgi?id=570553 👎🏻